README.md

```diff
@@ -1 +1,50 @@
-# custom-enumerable
\ No newline at end of file
+# Custom Enumerable
+
+## Description
+
+> This project is about learning how to use a module inside a class.
+
+## Built With
+
+- Ruby
+
+## Getting Started
+
+To get the content of this project locally you need to run these commands in your terminal.
+Before starting the steps below, be sure you have [Git](https://www.linode.com/docs/guides/how-to-install-git-on-linux-mac-and-windows/).
+1. Download the `Zip` or clone with `git clone https://github.com/Gegardus/custom-enumerable`
+2. `cd custom-enumerable`
```

Comment:
[OPTIONAL]

- In the README file, please mention the step on how to test the application. Hint: `ruby my_list.rb`.

Reply:
Thank you @Dipeshtwis for your approval and suggestion.